invoiceregistry.go

```diff
@@ -1009,7 +1013,7 @@ func (i *InvoiceRegistry) markSettleRequested(ctx context.Context,
 			"setTotal", setTotal,
 			"invoiceValue", state.invoice.Value)
 
-		return errors.New("set no longer complete")
+		return ErrSetNoLongerComplete
```

Comment:
I sort of forgot about this case where some of the htlcs have been released, but not all. Would be good to also convert this error to a grpc error in `server.SettleInvoice`.

Reply:
This should be addressed in https://github.com/bottlepay/lnmux/pull/18/commits/31d4088731b5a33ed0332e6f8f0b05b493848fbb now